cmd/utils/utils.go

```diff
@@ -597,3 +609,48 @@ func ethdkg(logger *logrus.Entry, eth interfaces.Ethereum, cmd *cobra.Command, a
 
 	return 0
 }
+
+func arbitraryethdkg(logger *logrus.Entry, eth interfaces.Ethereum, cmd *cobra.Command, args []string) int {
```

Comment:
Why is this all commented out?

Reply:
We created this CLI command for testing purposes. It's commented out because the function that it depends on it's not present in the current version of the smart contracts. I don't think it's required for now, let me remove this.